packages/core/src/browser/tree/tree-widget.ts

```diff
@@ -67,6 +73,15 @@ export const defaultTreeProps: TreeProps = {
     leftPadding: 16
 };
 
+/**
+ * Bare minimum common interface of the keyboard and the mouse event with respect to the key maskings.
+ */
+export interface ModifierAwareEvent {
```

Comment:
please put under `TreeWidget` namespace or maybe move to `BaseWidget`, it sounds very generic